sqlmesh/core/model/definition.py

```diff
@@ -595,6 +595,7 @@ def _statement_renderer(self, expression: exp.Expression) -> ExpressionRenderer:
                 only_execution_time=False,
                 default_catalog=self.default_catalog,
                 model_fqn=self.fqn,
+                raw_code=self._raw_code,
```

Comment:
I don't know if we need this interface. Can we just pass `self` instead of both `model_fqn` and `raw_code`? The builtins module can just use the model reference directly to extract the raw query